CMakeLists.txt

```diff
@@ -35,7 +35,10 @@ set(CMAKE_FORTRAN_EXTENSIONS OFF)
 # Location of .pycodestyle for norm checking within IODA-converters
 set( IODACONV_PYLINT_CFG_DIR ${CMAKE_CURRENT_SOURCE_DIR} )
 
-find_package(Python3 REQUIRED COMPONENTS Interpreter Development)
+# Python-finding settings
+set(Python3_FIND_REGISTRY "LAST")
```

Comment:
do we support windows or any microsoft platform?

Reply:
Since IODA was just merged with this line included, I'll do the same for ioda-converters.

Reply:
no need, just asking the question.